src/main/scala/org/constellation/primitives/storage/AcceptedTransactionService.scala

```diff
@@ -5,8 +5,8 @@ import org.constellation.primitives.{TransactionCacheData, TransactionSerialized
 import scala.collection.mutable
 
 
-class AcceptedTransactionService(size: Int = 50000)
-    extends StorageService[TransactionCacheData](size) {
+class AcceptedTransactionService()
```

Comment:
hm I've forgotten to remove that at all. it's inside `TransactionService` right now. same for `tx hash store`. I'll clean it up tomorrow